server/routes/overview/mappers/prisonerSupportNeedsMapper.test.ts

```diff
@@ -1,45 +1,43 @@
 import { parseISO, startOfDay } from 'date-fns'
 import type { PrisonerSupportNeeds } from 'viewModels'
-import type { LearnerProfile } from 'curiousApiClient'
 import toPrisonerSupportNeeds from './prisonerSupportNeedsMapper'
+import {
```

Comment:
This mapper test has changed a lot, but thats because the impl has done (see below)